lib/api.js

```diff
@@ -16,13 +15,15 @@ if (config.env === 'development') {
   const build = require('mastarm/lib/build')
   const util = require('mastarm/lib/util')
 
-  build({
-    config: config._defaultConfig,
-    files: util.parseEntries(config.entries),
-    watch: true
-  })
-  .then(() => console.log('JavaScript & CSS built..'))
-  .catch((err) => console.error('Error building JavaScript & CSS...', err))
+  if (process.argv.indexOf('no-build') === -1) {
```

Comment:
Not asking to remove this but in what case would you run this in development and not want to build?